lua/entities/gmod_wire_expression2/core/e2lib.lua

```diff
@@ -47,17 +47,12 @@ function E2Lib.setAng(ent, ang)
 	return ent:SetAngles(ang)
 end
 
---Blacklist format:
---<top folder the material is in>[%./\\]+<material name>
---Should prevent work-arounds like pp/./copy pp/./././copy pp\\copy etc.
 local material_blacklist = {
-	"pp[%./\\]+copy"
+	["pp/copy"] = true
 }
 local function validMaterial(material)
-	local lower = string.lower(material)
-	for _, v in ipairs(material_blacklist) do
-		if string.find(lower, v) then return "" end
-	end
+	local path = string.StripExtension(string.GetNormalizedFilepath(string.lower(material)))
+	if material_blacklist[path] then return "" end
```

Comment:
ew my eyes. Could you change this to `return material_blacklist[path] and "" or material`
Thanks.